packages/flame/test/components/component_test.dart

```diff
@@ -200,6 +200,56 @@ void main() {
       );
     });
   });
+
+  group('Component descendants', () {
```

Comment:
```suggestion
  group('descendants', () {
```

Should already be under the `group('Component')`.

Reply:
This group needs to be moved inside the "Component" group, so that every top-level group name corresponds to the name of the class being tested.

Reply:
@st-pasha is correct. That's exactly what I meant, sorry for the ambiguity.

Reply:
What do you think of this test group structure?
```dart
group('ClassName.method -', () {
  testWithFlameGame(
    'testName',
    (game) async {
      // Setup

      // Action

      // Result
    },
  );
});
```

Output will be: 
`ClassName.method - testName`
This will visually show which method is currently being tested.

Reply:
I prefer having:

```
group('ClassName', () {
  group('method1', () {});
  group('method2', () {});
});
```

Notice how this way we don't repeat the "ClassName".

Reply:
done